tests/foreman/data/zoo_custom_gpgkey.txt

```diff
@@ -0,0 +1,30 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
Is it good to give single Hard-coded GPG key? Cant we generate the GPG key from anywehre ?
Just a thought.